tests/HandleIframeAsTextTest.hack

```diff
@@ -0,0 +1,130 @@
+/* Created by Jake Polacek on 01/20/2022 */
+
+namespace HTMLPurifier\_Private\Tests;
+
+use function Facebook\FBExpect\expect;
+use type Facebook\HackTest\HackTest;
+use namespace HTMLPurifier;
+use namespace HTMLPurifier\{Strategy, Token, Lexer, Enums};
+
+class HandleIframeAsTextTest extends HackTest {
+	public function testSingleNestedElement(): void {
+		echo "\ntestSingleNestedElement()...";
+		//porting over first config classes....
+		$config = HTMLPurifier\HTMLPurifier_Config::createDefault();
+		$policy = HTMLPurifier\HTMLPurifier_Policy::fromDefault()
+			|> $$->addAllowedTag(Enums\HtmlTags::IFRAME);
+		$purifier = new HTMLPurifier\HTMLPurifier($config, $policy);
+		$dirty_html = '<iframe src="https://example.org" title="iframe Example 1" width="400" height="300">
+	<b></b>
+</iframe>';
+		$clean_html = $purifier->purify($dirty_html);
+		expect($clean_html)->toEqual(
+			'<iframe src="https://example.org" title="iframe Example 1" width="400" height="300">
+	&lt;b&gt;&lt;/b&gt;
```

Comment:
Because `<b></b>` is handled as text, `<` and `>` are escaped, but we don't unescape it after sanitization. I don't think this should be an issue because I think that it's more likely than not the case that anyone using markdown inside of an iframe is being malicious and the markdown isn't even displayed, but I'm interested in hearing others' thoughts on this.